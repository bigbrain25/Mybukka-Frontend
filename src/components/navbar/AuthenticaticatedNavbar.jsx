import React from 'react';
import Brand from 'Components/brand/Brand';
import UserDefaultImage from './UserDefaultImage';
import './navbar.scss';

const AuthenticaticatedNavbar = () => (
  <div className="container">
    <nav className="navbar navbar-light">
      <Brand />
      <div className="form-inline">
        <UserDefaultImage />
      </div>
    </nav>
  </div>
);

export default AuthenticaticatedNavbar;
