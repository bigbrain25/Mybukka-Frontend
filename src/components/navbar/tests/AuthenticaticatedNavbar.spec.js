import React from 'react';

import AuthenticaticatedNavbar from '../AuthenticaticatedNavbar';

describe('Authenticaticated Navbar', () => {
  const wrapper = mount(<AuthenticaticatedNavbar />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
