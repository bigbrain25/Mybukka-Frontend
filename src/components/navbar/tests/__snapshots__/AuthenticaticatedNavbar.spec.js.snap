// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Primary Navbar renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Provider
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  >
    <MemoryRouter>
      <Connect(BukkaAuthenticatedNav)
        status={
          Object {
            "authenticated": true,
          }
        }
      />
    </MemoryRouter>
  </Provider>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <MemoryRouter>
        <Connect(BukkaAuthenticatedNav)
          status={
            Object {
              "authenticated": true,
            }
          }
        />
      </MemoryRouter>,
      "value": Object {
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "storeState": Object {
          "authenticationReducer": Object {
            "errorMessage": "",
            "status": Object {
              "authenticated": false,
              "error": false,
            },
            "user": Object {},
          },
          "deliveryModeReducer": Object {
            "mode": "pickup",
          },
          "homeReducer": Object {
            "type": "Sign Up",
          },
        },
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Connect(BukkaAuthenticatedNav)
          status={
            Object {
              "authenticated": true,
            }
          }
        />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "status": Object {
            "authenticated": true,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
    "type": Object {
      "$$typeof": Symbol(react.provider),
      "_context": Object {
        "$$typeof": Symbol(react.context),
        "Consumer": Object {
          "$$typeof": Symbol(react.context),
          "_calculateChangedBits": null,
          "_context": [Circular],
        },
        "Provider": [Circular],
        "_calculateChangedBits": null,
        "_currentRenderer": null,
        "_currentRenderer2": null,
        "_currentValue": null,
        "_currentValue2": null,
        "_threadCount": 0,
      },
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <MemoryRouter>
          <Connect(BukkaAuthenticatedNav)
            status={
              Object {
                "authenticated": true,
              }
            }
          />
        </MemoryRouter>,
        "value": Object {
          "store": Object {
            "clearActions": [Function],
            "dispatch": [Function],
            "getActions": [Function],
            "getState": [Function],
            "replaceReducer": [Function],
            "subscribe": [Function],
          },
          "storeState": Object {
            "authenticationReducer": Object {
              "errorMessage": "",
              "status": Object {
                "authenticated": false,
                "error": false,
              },
              "user": Object {},
            },
            "deliveryModeReducer": Object {
              "mode": "pickup",
            },
            "homeReducer": Object {
              "type": "Sign Up",
            },
          },
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Connect(BukkaAuthenticatedNav)
            status={
              Object {
                "authenticated": true,
              }
            }
          />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "status": Object {
              "authenticated": true,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      "type": Object {
        "$$typeof": Symbol(react.provider),
        "_context": Object {
          "$$typeof": Symbol(react.context),
          "Consumer": Object {
            "$$typeof": Symbol(react.context),
            "_calculateChangedBits": null,
            "_context": [Circular],
          },
          "Provider": [Circular],
          "_calculateChangedBits": null,
          "_currentRenderer": null,
          "_currentRenderer2": null,
          "_currentValue": null,
          "_currentValue2": null,
          "_threadCount": 0,
        },
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
